REQUIRE

```diff
@@ -1,2 +1,2 @@
-julia 0.7-
-BinaryProvider 0.3.0
+julia 0.7
+BinaryProvider 0.4.0
```

Comment:
This will have to be 0.4.1, once https://github.com/JuliaPackaging/BinaryProvider.jl/pull/97 is merged.

Reply:
Need to change this?